tripleo-ciscoaci/deployment/opflex/opflex-agent-container-puppet.yaml

```diff
@@ -148,6 +148,50 @@ parameters:
        The starting backoff value for retrying of policy requests, in seconds. The
        backoff is exponential, starting at this value, and continuing up to 16 retries.
     type: number
+  OpflexStatisticsMode:
+    default: 'real'
+    type: string
+  OpflexEnabledStatisticsInterface:
+    default: true
+    type: boolean
+  OpflexIntervalStatisticsInterface:
+    default: 30000
+    type: number
+  OpflexEnabledStatisticsContract:
+    default: true
+    type: boolean
+  OpflexIntervalStatisticsContract:
+    default: 10000
+    type: number
+  OpflexEnabledStatisticsSecurityGroup:
+    default: true
+    type: boolean
+  OpflexIntervalStatisticsSecurityGroup:
+    default: 10000
+    type: number
+  OpflexDisabledStatisticsServiceFlow:
+    default: false
+    description: Disable/Enable stats flow creation
+    type: boolean
+  OpflexEnabledStatisticsService:
+    default: true
+    description: Disable/Enable stats collection
+    type: boolean
+  OpflexIntervalStatisticsService:
+    default: 10000
+    type: number
+  OpflexEnabledStatisticsTableDrop:
+    default: true
+    type: boolean
+  OpflexIntervalStatisticsTableDrop:
+    default: 10000
```

Comment:
As per https://github.com/noironetworks/opflex/blob/master/agent-ovs/ovs/OVSRenderer.cpp#L512, this should be 30000